src/components/ValuePicker/types.ts

```diff
@@ -0,0 +1,58 @@
+import type {RadioItem} from '@components/SelectionList/types';
+
+type ValuePickerItem = RadioItem & {
```

Comment:
Do we really need `RadioItem` props in `ValuePickerItem` type? Or is it just to fit the typing of `SelectionList`? If the second, then maybe it makes sense to try to update SelectionList typing instead

Reply:
Yeah, it can be done much easier! Fixed and tested SelectionList in a few places and it works fine